<!--
import data_analytics.github as github
print(github.create_jupyter_notebook_header("marcocrowe", "learn-mathematical-methods", "exam-2024-mathematical-methods-2-math06100-solution.ipynb", "master"))
-->
<table style="margin: auto;"><tr><td><a href="https://mybinder.org/v2/gh/marcocrowe/learn-mathematical-methods/master?filepath=exam-2024-mathematical-methods-2-math06100-solution.ipynb" target="_parent"><img src="https://mybinder.org/badge_logo.svg" alt="Open In Binder"/></a></td><td>online editors</td><td><a href="https://colab.research.google.com/github/marcocrowe/learn-mathematical-methods/blob/master/exam-2024-mathematical-methods-2-math06100-solution.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a></td></tr></table>


# Mathematical Methods 2 - Exam Paper - 2023 - Summer

## Setup

In [4]:
from sympy import diff, exp, cos, latex, ln, sin, solve, symbols
from IPython.display import display, Markdown

def calculate_and_display_derivative(title, x, x_equation, y_name="y"):
    dx_dy = diff(x_equation, x)
    # Display the derivative in LaTeX format
    display(Markdown(f"### {title}"))
    display(Markdown("$\Large\n" + y_name + " = " + latex(x_equation) + r" \implies \\ \frac{d" + y_name + "}{d" + x.name + "} = " + latex(dx_dy) + "$"))

## Instructions


**Time Allowed**: 2 hours

**Instructions**:  Answer 4 questions. All questions carry equal marks

The use of programmable or text storing calculators is expressly forbidden.

Please note that where a candidate answers more than the required number of questions, the examiner will mark all questions attempted and then select the highest scoring ones.

***Requirements for this paper:***

1. ***Log Tables***
2. ***Calculators***


## Question 1 (Total Marks 25)

### Question 1(a) [12 Marks]

Solve the triangle shown below:

![Figure 1](images/exam-2023-mathematical-methods-2-math06072-paper/figure-1.png)

### Question 1(b) [3 Marks]

When the angle of elevation of the sun is 10°, an upright flagpole casts a shadow of length of 20m. Calculate the height of the pole.

### Question 1(c) [4 Marks]



Given $2.3\ln{\dfrac{1.3}{x^2}}=9.1$, Solve for x:

#### Question 1(c) Answer

<details>
<summary>A Note on ln</summary>
What is the difference between ln and log?

$\ln$ is the natural logarithm i.e. $$\ln{x} = \log_{e}{x}$$

So for any of the log rules, e.g the quotient rule:  
$$\log_{c}{\dfrac{a}{b}} = \log_{c}{a} - \log_{c}{b}$$  
for natural logarithms, substitute $c=e$ to get the natural logarithm rule for quotients:  
$$\log_e{\dfrac{a}{b}} = \log_e{a} - \log_e{b}$$  
and $\log_e{x} = \ln{x}$ so the rule becomes:  
$$\ln{\dfrac{a}{b}} = \ln{a} - \ln{b}$$

---
</details>

$2.3\ln{\dfrac{1.3}{x^2}}=9.1$

The log quotient rule $\ln{\dfrac{a}{b}} = \ln{a} - \ln{b}$ applies to $\ln{\dfrac{1.3}{x^2}}$  
$\therefore \ln({\dfrac{1.3}{x^2}}) = \ln({1.3}) - \ln({x^2})$

$2.3(\ln({1.3}) - \ln({x^2}))=9.1$  
$2.3\ln({1.3}) - 2.3\ln({x^2})=9.1$

The power rule for logarithms states that $\log_{b}{x^n} = n\log_{b}{x}$
$\therefore \ln({x^2}) = 2\ln({x})$

$2.3\ln({1.3}) - 2.3(2\ln({x}))=9.1$  
$2.3\ln({1.3}) - 4.6\ln({x})=9.1$  
$2.3\ln({1.3}) - 9.1=4.6\ln({x})$  
$\dfrac{2.3\ln({1.3}) - 9.1}{4.6}=\ln({x})$  
$\ln({x}) = \dfrac{2.3\ln({1.3}) - 9.1}{4.6}$

Inverse Property of Logarithms and Exponentials: If $ln(a)=b$, then $a=e^b$.  
This means that for any real number $k$, $ln(x)=k$ is equivalent to $x=e^k$.

Let $k=\dfrac{2.3\ln({1.3}) - 9.1}{4.6}$
$ln(1.3) \approx 0.2624$

$k \approx \frac{2.3 \times 0.2624 - 9.1}{4.6} \approx \frac{0.6048 - 9.1}{4.6} \approx \frac{-8.4952}{4.6} \approx -1.8470$

$x = e^{\dfrac{2.3\ln({1.3}) - 9.1}{4.6}}$

$\therefore ln(x)\approx k \approx -1.8470$
$\therefore x=e^{-1.8470} \approx 0.157$

$\boxed{x \approx 0.157}$

In [5]:
# Define the variable
x = symbols('x')

# Define the equation
equation = 2.3 * ln(1.3 / x**2) - 9.1

# Isolate the logarithm term
isolated_log = solve(equation, x)

isolated_log


[-0.157697168944812, 0.157697168944812]

The SymPy library provides two solutions to the equation

However, since the logarithm of a negative number is not defined in the real number system, the valid solution is:

$ x \approx 0.1577 $

### Question 1(d) [6 Marks]

For the function shown below, determine the second order differential $f(x)$.

$f(x) =-6x+\frac{1}{\sqrt[3]{x}} - \frac{1}{4x^3}$